gitapp/lib/users_list/User_card.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:gitapp/constants/constants.dart';
+import 'package:gitapp/user_details/user_details.dart';
+
+class UserCard extends StatelessWidget {
+  UserCard(
+      {required this.user,
```

Comment:
I think that we could pass only an User object which could contain the UserName, and avatar image.

Reply:
Solved